Workbooks/Log Analytics - Performance Analysis/Performance Analysis for a group of VMs/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"VM Performance Analysis",
+    "name":"Performance Analysis for a group of VMs",
```

Comment:
should group be upper case to match with above?

Reply:
Would you also rename the file to match as well?

Reply:
Thanks.